hudi-common/src/main/java/org/apache/hudi/common/model/HoodieKey.java

```diff
@@ -27,13 +32,13 @@
  * - recordKey : a recordKey that acts as primary key for a record.
  * - partitionPath : the partition path of a record.
  */
-public class HoodieKey implements Serializable {
+public class HoodieKey implements Serializable, KryoSerializable {
```

Comment:
We have used kryo as the serializer of shuffle. I wonder that Why we explicitly specify KryoSerializable? Same to HoodieRecord.

Reply:
Correct. We want to enforce that

 - `HoodieRecord` are serializable by Kryo
 - That we control serialization sequence